Automattic/newspack-theme
# Fix: Update screenshots for Newspack Nelson, Joseph, to correct logo

### All Submissions:






### Changes proposed in this Pull Request:

This change updates the screenshots for Newspack Nelson and Joseph, to correct the out of date Newspack logo used.

I also ran them both through TinyPNG and will do the rest of the themes, to make the file size less huge. 

Closes #716.

### How to test the changes in this Pull Request:

1. Apply PR.
2. Review screenshots for the two themes; confirm that they're now using the version with the 'corrected' N (for lack of a better way to put it!), like on https://newspack.blog/.

### Other information:
